pkg/endpoint/bpf.go

```
@@ -155,11 +155,12 @@ func (e *Endpoint) writeInformationalComments(w io.Writer) error {
 
 	fw.WriteString("/*\n")
 	fw.WriteString(" * Labels:\n")
-	if e.SecurityIdentity != nil {
-		if len(e.SecurityIdentity.Labels) == 0 {
+	securityIdentity, _ := e.GetSecurityIdentity()
```

Errors must not be ignored like this.

---

I would like to confirm if it is ok to return only security identity in GetSecurityIdentity. If I remove lock in GetSecurityIdentity, I think that we might not need to return an error.

I also want suggestions from you @nathanjsweet .

Thanks in advance.